### PR TITLE
[JS] Llama 3 + function calling + E2B Code interpreter example

### PR DESCRIPTION
**Pull request by jakubno**
JavaScript version of llama 3 example with code interpreter capabilites

### CODE REVIEW

File: examples/llama-3-code-interpreter-js/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Llama 3 + function calling + E2B Code interpreter        
+**Powered by open-source [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by [E2B](https://e2b.dev/docs)**
+
+E2B's code interpreter SDK quickly creates a secure cloud sandbox powered by [Firecracker](https://github.com/firecracker-microvm/firecracker).
+
+Inside this sandbox is a running Jupyter server that the LLM can use.
+
+### Step 1: Install dependencies
+
+We start by install the [E2B code interpreter SDK](https://github.com/e2b-dev/code-interpreter) and [Groq's JS SDK](https://github.com/groq/groq-typescript).
```
Comment (tizkovatereza) [inline examples/llama-3-code-interpreter-js/README.md:10]: "install" -> "installing"

Comment (tizkovatereza) [inline examples/llama-3-code-interpreter-js/README.md:10]: Code Interpreter SDK is with capital letters

File: examples/llama-3-code-interpreter-js/.env.template [hunk 2]
```diff
@@ -0,0 +1,2 @@
+GROQ_API_KEY=
```
Comment (tizkovatereza) [inline examples/llama-3-code-interpreter-js/.env.template:1]: Would add links, something like

```python
# TODO: Get your E2B API key from https://e2b.dev/docs/getting-started/api-key
E2B_API_KEY=""

# TODO: Get your Groq API key from https://console.groq.com/keys
GROQ_API_KEY=""
```

Comment (jakubno) [reply examples/llama-3-code-interpreter-js/.env.template:1]: Thanks!

**Pull request merged**